src/routes/Header.svelte

```diff
@@ -2,16 +2,54 @@
   import { page } from "$app/stores";
   import logo from "$lib/images/svelte-logo.svg";
   import github from "$lib/images/github.svg";
+  import "iconify-icon";
+
+
+  $: current = $page.url.pathname
+
 </script>
 
-<header>
-  <div class="corner">
-    <a href="https://kit.svelte.dev">
-      <img src={logo} alt="SvelteKit" />
-    </a>
-  </div>
+<header class="navbar bg-base-100">
+    <div class="navbar-start">
+      <a href='/' class="btn btn-ghost normal-case text-xl">Planet Of The Bugs</a>
+    </div>
+
```

Comment:
Can you put the `ul` inside a `nav` element? We should aim to write semantic HTML.

Reply:
will look into it

Reply:
yeah its a common practice